### PR TITLE
style: small updates to the Picket integration docs

### PR DESCRIPTION
**Pull request by devstein**
## What kind of change does this PR introduce?

Docs update to keep them in sync with https://docs.picketapi.com/picket-docs/reference/integrations/supabase#overview

## What is the current behavior?

N/A

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by kiwicopple — APPROVED** [review]

File: apps/docs/pages/guides/integrations/picket.mdx [hunk 1]
```diff
@@ -72,7 +72,7 @@ Click `Save` to create the new table.
 Now we want to make sure that only the `todos` owner, the user's `wallet_address`, can access their todos. The key component of the this RLS policy is the expression
 
 ```sql
-((jwt() ->> 'walletAddress'::text) = wallet_address)
+((auth.jwt() ->> 'walletAddress'::text) = wallet_address)
 ```
```
Comment (kiwicopple) [inline apps/docs/pages/guides/integrations/picket.mdx:76]:
```suggestion
```

```

File: apps/docs/pages/guides/integrations/picket.mdx [hunk 2]
```diff
@@ -72,7 +72,7 @@ Click `Save` to create the new table.
 Now we want to make sure that only the `todos` owner, the user's `wallet_address`, can access their todos. The key component of the this RLS policy is the expression
```
Comment (kiwicopple) [inline apps/docs/pages/guides/integrations/picket.mdx:73]:
```suggestion

```

File: apps/docs/pages/guides/integrations/picket.mdx [hunk 3]
```diff
@@ -70,9 +70,8 @@ Click `Save` to create the new table.
 ### Setup Row Level Security (RLS)
 
 Now we want to make sure that only the `todos` owner, the user's `wallet_address`, can access their todos. The key component of the this RLS policy is the expression
```
Comment (kiwicopple) [inline apps/docs/pages/guides/integrations/picket.mdx:72]:
```suggestion
Now we want to make sure that only the `todos` owner, the user's `wallet_address`, can access their todos. The key component of the this RLS policy is the expression

```

**Comment by devstein** [comment]
Thanks for the review and changes @kiwicopple

**Pull request merged**